organization/_posts/2018/2018-07-19-coordination.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "HSF Weekly Meeting #142, 18 July, 2018"
+layout: default
+---
+
+# {{page.title}}
+
+#### *Present/Contributors*: Graeme Stewart, Andrea Valassi, Dario Menasce, Benedikt Hegner
```

Comment:
Add "Eduardo Rodrigues". There is probably 1-2 persons missing, connecting from outside.